iFixit/react-commerce
# Bump shopify api version to 2024-01

[2024-01 release notes](https://shopify.dev/docs/api/release-notes/2024-01#storefront-api-changes). There were no changes to the generated SDK when I ran `pnpm dev`.

Connects https://github.com/iFixit/ifixit/pull/51793

## QA

Visit the Vercel preview and make sure we can still view product pages with `disableCacheGets` and initiate checkout from the mini cart.
